tests/validators/array.test.ts

```diff
@@ -111,4 +123,23 @@ describe('ArrayValidator', () => {
 			});
 		});
 	});
+
+	describe('Clone', () => {
+		// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+		// @ts-expect-error
```

Comment:
```suggestion
		// @ts-expect-error Accessing protected method
```

I think? Anyway if you add a reason then you don't need the eslint ignore.